front/app/modules/commercial/matomo/actions.ts

```diff
@@ -0,0 +1,73 @@
+// routes
```

Comment:
Extracted this stuff from `matomo/index` just to organize things a bit better and make it easier to test our code